<launch>

   <master auto="start"/>
 <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(find lidart_pure_pursuit)/maps/nikhil_processed.yaml"/>
    <!-- <param name="base_global_planner" value="global_planner/GlobalPlanner"/> -->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_global_planner" value="rrt_planner/GlobalPlanner"/>
    <rosparam file="$(find lidart_2dnav)/launch/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find lidart_2dnav)/launch/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find lidart_2dnav)/launch/local_costmap_teb_params.yaml" command="load" />
    <rosparam file="$(find lidart_2dnav)/launch/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find lidart_2dnav)/launch/base_local_planner_teb_params.yaml" command="load" />
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <remap from="odom" to="/pf/pose/odom" />
    <param name="controller_frequency" value="0.25" />
    </node>

</launch>
